eslzArm/README.md

```diff
@@ -0,0 +1,201 @@
+# Enterprise-Scale Landing Zones ARM templates
+
+This folder contains the first-party ARM templates for Enterprise-Scale which and are being used when deploying and bootstrapping in the Azure Portal.
+
+For customers who cannot deploy via portal, but rather want to clone the repository and sequence the deployments on their own, they can follow the instructions below.
+
+> **Note:** There's a strict sequencing required in order to achieve the same outcome as when deploying via the Azure portal, and any modification and changes to the templates are not supported.
+
+## Do-It-The-Hard-Way deployment instructions for Enterprise-Scale using Azure PowerShell
```

Comment:
Not sure we can call it this 😁

Reply:
How about creating a github workflow instead? I know it's off the wall, but if we're preaching IaC and CI/CD maybe we should provide an alternative to the portal/commandline option.

Reply:
@jfaurskov, our answer - and recommendation fro IaC and CI/CD is through AzOps. More information about that @ https://github.com/Azure/Enterprise-Scale/wiki/Deploying-Enterprise-Scale#operating-the-azure-platform-using-azops-infrastructure-as-code-with-github-actions